open_ce/build_env.py

```diff
@@ -122,6 +122,17 @@ def build_env(args):
                                conda_build_config=args.conda_build_config,
                                packages=inputs.parse_arg_list(args.packages))
 
+    #import matplotlib.pyplot as plt
```

Comment:
Should this code be removed? Or added as some sort of debugging function?

Reply:
Thanks!
I think we might be able to add it as some rendering function in the future.